packages/html-reporter/src/labelUtils.tsx

```diff
@@ -25,7 +25,7 @@ export function escapeRegExp(string: string) {
 
 // match all tags in test title
 export function matchTags(title: string): string[] {
-  return title.match(/@(\w+)/g)?.map(tag => tag.slice(1)) || [];
+  return title.match(/@([\w-]+)/g)?.map(tag => tag.slice(1)) || [];
```

Comment:
what about underscore?

Reply:
`\w` - matches any word characters, including underscore `_`
`-` - is exceptional character what using somebody for tags
`# $ % ^ & * + = < > |` - are more special symbols, and i think what almost nobody not using it for tags
If tag will containing special symbol it should be escaped by search engine and no longer work

Reply:
What is the conceptual difference between `-` and `+` when it comes to the tab value, why do you want to treat them differently?

Reply:
`[\w-]+` - it means
`\w` - any one of [a-zA-Z0-9_]
`-` - one of `-`
`+` - as much as possible
It match like `@CBC-123`, `@flaky`
Or I'm not understand you what you want)
I offered such a solution to the "problem", you have a different vision of it.  we still won't be able to cover all possible options) but we should try to do so.  and it is necessary to exclude options that, on the contrary, can harm.

Reply:
I am suggesting to be more lax in matching the tags. Clearly we have already underestimated the expressiveness of the tags, hence this follow up. So no considerations like "more special" stand at this point. I'm thinking about tags like:

`@smoke:p1`
`@issue[123]`
`@issue#123`
`@$$$`
`@tl/dr`

etc.

Hence my question of why not treat any uninterrupted sequence as a tag and minimize the number of follow up patches like this one.

Reply:
(backslash escaping space-separated tags would also be fine)

Reply:
i'm try using `escapeRegExp` when matching tags from test title but after they looks like `@issue\[123\]`, `@\$\$\$` and after be clicked this string not matching anything


https://user-images.githubusercontent.com/11705902/235366845-6139f600-8045-41f6-9c8b-72d6ea68a96a.mp4